src/remodel_api/remodel.rs

```diff
@@ -282,11 +332,12 @@ impl Remodel {
         Remodel::import_tree_root(context, source_tree)
     }
 
-    fn write_existing_model_asset(
+    fn write_model_asset(
         context: Context<'_>,
         lua_instance: LuaInstance,
         asset_id: u64,
-    ) -> rlua::Result<()> {
+        queries: &[(&str, &str)],
```

Comment:
I don't think this is the right interface here. We should instead accept a struct of known fields that are turned into the correct querystring parameters by this function.